PRIVACY.md

```diff
@@ -1,22 +1,53 @@
-# Semgrep Privacy Policy
+# Semgrep CI Privacy Policy
 
-Semgrep collects usage data to provide useful results and to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
+## Metrics
 
-## Scan data
+Semgrep CI (also known as semgrep-action or semgrep-agent) may collect non-identifiable aggregate metrics to help improve the product. This document describes:
+
+- the principles that guide our data-collection decisions
+- the breakdown of the data that are and are not collected
+- how to opt-out of Semgrep CI’s metrics
+
+## Principles
+
+These principles inform our decisions around data collection:
+
+1. *Transparency*: Collect and use data in a way that is clearly explained to the user and benefits them
+2. *User control*: Put users in control of their data at all times
+3. *Limited data*: Collect what is needed, de-identify where possible, and delete when no longer necessary
+
+## Collected data
+
+Semgrep CI collects opt-out non-identifiable aggregate metrics for improving the user experience, guiding Semgrep feature development, and identifying regressions. It relies on Semgrep CLI’s metric collection, which is discussed in detail in that project’s [PRIVACY.md](https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md).
+
+## Opt-out behavior
+
+Semgrep CI’s metrics can be disabled by setting the environment variable SEMGREP_SEND_METRICS=0 or using the flag --disable-metrics. If this environment variable or flag is not set, aggregate metrics are enabled.
```

Comment:
```suggestion
Semgrep CI’s metrics can be disabled by setting the environment variable `SEMGREP_SEND_METRICS=0` or using the flag `--disable-metrics`. If this environment variable or flag is not set, aggregate metrics are enabled.
```

Reply:
@chmccreery We'll need to update this section with whatever environment variable and flag names you choose.

Reply:
I changed it to SEMGREP_SEND_METRICS to match the Semgrep CLI name 😄